### PR TITLE
license_id has been renamed licenseid (not a FKEY field)

### PR DESCRIPTION
**Pull request by trasher**

### CODE REVIEW

**Comment by trasher** [comment]
This fix is real and do pass the tests.
Remaining issues are once again specific to unit tests ; out of the scope of this pull request.

File: phpunit/bootstrap.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
 define('TU_USER', '_test_user');
 define('TU_PASS', 'PhpUnit_4');
 
-global $CFG_GLPI;
+global $CFG_GLPIi, $GLPI_CACHE;
```
Comment (orthagh) [inline phpunit/bootstrap.php]: vim insertion mode :D

Comment (trasher) [reply phpunit/bootstrap.php]: Good catch :1st_place_medal:

**Comment by ddurieux** [comment]
Fixed in #2740

**Pull request closed without merging**